docs/src/@primer/gatsby-theme-doctocat/nav.yml

```diff
@@ -165,3 +165,7 @@
       url: /FormGroup
     - title: SelectMenu
       url: /deprecated/SelectMenu
+    - title: ActionList
```

Comment:
Can we move these to the top of the list please? The lack of alphabetical ordering is a bit more noticeable now. Might need to fix the others too 🙏 
![Screenshot 2022-02-25 at 17 20 15](https://user-images.githubusercontent.com/13340707/155759419-f2c7bb05-c726-4870-b02f-587795603729.png)